lib/start.js

```diff
@@ -148,7 +148,7 @@ const start = (passthroughArgs, buildConfig = config.defaultBuildConfig, options
         }
         if (url.startsWith('http') && url.includes('.')) {
           const found = whitelistedUrlPrefixes.find((prefix) => {
-            return url.startsWith(prefix)
+            return RegExp('^' + prefix).test(url)
```

Comment:
Actually this creates an issue where the `.`'s (and possibly other regex special characters?) in whitelistedUrlPrefixes need to be escaped or else they don't work like they did before.

For instance `RegExp('https://safebrowsing.googleapis').test('https://safebrowsingdgoogleapis')` would return true

Instead of modifying all the entries to escape special characters, i recommend just creating a separate list which is regexes instead of exact strings.

Reply:
Oh yeah! great catch, updating